codestyle/checkstyle-suppressions.xml

```diff
@@ -29,23 +29,50 @@
   <!-- Code copied from TestNG to apply a bugfix -->
   <suppress checks="AvoidStaticImport" files="[\\/]org[\\/]testng[\\/]" />
 
+  <suppress checks="LocalFinalVariableName" files="[\\/]target[\\/]generated-sources[\\/]" />
```

Comment:
Remove all rules from this file for generated-source and generated-test-sources and add the following two rules:

  <suppress checks=".*" files="[\\/]target[\\/]generated-test-sources[\\/]" />
  <suppress checks=".*" files="[\\/]target[\\/]generated-sources[\\/]" />

Reply:
The URL in Line 27 of this file is out of date. Please also change it to: https://checkstyle.org/filters/suppressionfilter.html

Reply:
good catch, fixed, Thanks